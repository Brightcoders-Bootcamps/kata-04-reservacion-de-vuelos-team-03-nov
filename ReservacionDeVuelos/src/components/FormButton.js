import React from 'react';
import {
    Text,
    TouchableOpacity
} from 'react-native';


const FormButton = () => {
    return(
        <TouchableOpacity>
            <Text>Hey, it works!</Text>
        </TouchableOpacity>
    );
};

export default FormButton;
