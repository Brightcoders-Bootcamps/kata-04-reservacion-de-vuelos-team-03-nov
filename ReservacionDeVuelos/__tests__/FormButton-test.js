import React from 'react';
import renderer from 'react-test-renderer';
import FormButton from '../src/components/FormButton';

test('renders correctly', () => {
  const tree = renderer.create(<FormButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
