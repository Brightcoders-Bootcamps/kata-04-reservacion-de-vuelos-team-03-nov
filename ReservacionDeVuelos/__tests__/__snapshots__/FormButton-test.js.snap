// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": undefined,
      "borderRadius": 10,
      "height": 88.93333333333334,
      "justifyContent": "center",
      "marginTop": 10,
      "opacity": 1,
      "padding": 10,
      "width": "85%",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontSize": 16,
          "fontWeight": "bold",
        },
        Object {
          "color": undefined,
        },
      ]
    }
  />
</View>
`;
